frontend/public/components/resource-dropdown.tsx

```diff
@@ -90,63 +64,82 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
   const isKindSelected = (kind: string) => {
     return _.includes(selected, kind);
   };
-  // Create dropdown items for each resource.
-  const items = resources.map((model) => (
-    <DropdownItem
+
+  // Create select option for each resource.
+  const items = resources.toArray().map((model) => (
```

Comment:
Don't convert to an ordered map and back to an array. Just use an array when initializing `resources`